### PR TITLE
TF-6866 Update policy sets documentation

### PR DESCRIPTION
**Pull request by ker-an**
### What

Users can now enforce a policy set on all workspaces in a project. This PR updates the following pages:

- policy-enforcement/index.mdx
- manage-policy-sets.mdx
- opa/index.mdx
- opa/vcs.mdx
- sentinel/index.mdx
- sentinel/vcs.mdx
- users-teams-organizations/organizations.mdx

**Note:** The changes to the Policy Sets API documentation are in a separate PR: https://github.com/hashicorp/terraform-docs-common/pull/387

### Why
Currently, TFC users can apply policy sets to all current and future workspaces (globally) or individually selected workspaces in an organization. With the introduction of projects, users would like to apply policy sets to all current and future workspaces within projects without having to individually select each workspace.

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages. **N/A**
- [x] API documentation and the API Changelog have been updated. **N/A**
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate. **N/A**
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages. **N/A**
- [x] New pages have metadata (page name and description) at the top. **N/A**
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility. **N/A**
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars. **N/A**
- [x] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 1]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
-   - **(VCS Only) Policies Path**: Specify the sub-directory in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.
-   - **(Managed Policies Only) Policies**: Select managed policies to add to the policy set. You can only add policies written with the policy framework you selected for the policy set.
+1. Choose a **Policy framework** for the policies you want to add. You can only create a policy set from policies written using the same framework. You cannot change the framework type after creation.
+1. Choose a policy set scope:
+   - **Policies enforced globally:** Terraform Cloud will automatically enforce this global policy set on all existing and future workspaces.
```
Comment (Netra2104) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:114]: on all existing and future workspaces + and projects?

**Review by Netra2104 — APPROVED** [review]
Left one question, but other than that lgtm Keran 👍 !

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx (hunk 1 above)
Comment (hs26gill) [reply website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:114]: Yes, it is enforced on all the workspaces in the organization.
@ker-an Maybe we can add "in the organization" to be more explicit?

**Review by hs26gill — APPROVED** [review]
Left one suggestion, otherwise looks good

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx (hunk 1 above)
Comment (ker-an) [reply website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:114]: Yup! Great suggestion @hs26gill, I will add that bit.

File: website/docs/cloud-docs/policy-enforcement/index.mdx [hunk 2]
```diff
@@ -21,9 +21,9 @@ You can use two policy-as-code frameworks to define fine-grained, logic-based po
...
 
-You group policies into policy sets and apply those policy sets to one or more workspaces in your organization. For each run in those workspaces, Terraform Cloud checks the Terraform plan against the policy set. You can create policy sets directly in the Terraform Cloud UI, by connecting Terraform Cloud to your version control system, or through the Terraform Cloud API. A policy set must only contain policies written in a single policy framework (Sentinel or OPA), but you can add both Sentinel and OPA policy sets to each workspace.
+Policy sets are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces, in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. You can create policy sets directly in the Terraform Cloud UI, by connecting Terraform Cloud to your version control system, or through the Terraform Cloud API. A policy set must only contain policies written in a single policy framework (Sentinel or OPA), but you can add both Sentinel and OPA policy sets to each workspace.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/index.mdx:26]: Suggesting splitting this into two paragraphs for readability and adding some small tweaks for style.
```suggestion
Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. 

You can create policy sets using the [Terraform Cloud user interface](/terraform/cloud-docs/policy-enforcement/manage-policy-sets#create-policy-sets), the Terraform Cloud API, or by connecting Terraform Cloud to your version control system. A policy set can only contain policies written in a single policy framework (Sentinel or OPA). However, you can add Sentinel or OPA policy sets to the same workspace.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 3]
```diff
@@ -12,7 +12,7 @@ Policies are rules that Terraform Cloud enforces on Terraform runs. You can defi
 -> **Note:** Terraform Cloud **Free** Edition includes one policy set of up to five policies. In Terraform Cloud **Plus** Edition, you can connect a policy set to a version control repository or create policy set versions via the API. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
 <!-- END: TFC:only name:pnp-callout -->
 
-You group individual policies into policy sets and apply those policy sets to one or more workspaces in your organization. For each run in those workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop the run.
+Policy sets are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces, in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop the run.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:15]:
```suggestion
Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 4]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
-   - **(VCS Only) Policies Path**: Specify the sub-directory in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.
-   - **(Managed Policies Only) Policies**: Select managed policies to add to the policy set. You can only add policies written with the policy framework you selected for the policy set.
+1. Choose a **Policy framework** for the policies you want to add. You can only create a policy set from policies written using the same framework. You cannot change the framework type after creation.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:112]:
```suggestion
1. Choose a **Policy framework** for the policies you want to add. A policy set can only contain policies that use the same framework (OPA or Sentinel). You cannot change the framework type after creation.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 5]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
+1. Choose a policy set scope:
+   - **Policies enforced globally:** Terraform Cloud will automatically enforce this global policy set on all existing and future workspaces in the organization.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:114]:
```suggestion
   - **Policies enforced globally:** Terraform Cloud automatically enforces this global policy set on all of an organization's existing and future workspaces.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 6]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
+   - **Policies enforced on selected projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to enforce this policy set on. This affects all current and future workspaces for any selected projects.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:115]:
```suggestion
   - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 7]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
+1. **(OPA Only)** Allow **Overrides** where users with override policy permissions can let Terraform apply plans that have [mandatory policy](#policy-enforcement-levels) failures.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:116]:
```suggestion
1. **(OPA Only)** Allow **Overrides**, which enables users with override policy permissions to apply plans that have [mandatory policy](#policy-enforcement-levels) failures.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 8]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
+1. **(VCS Only)** Specify the **VCS branch** within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses the default branch of the VCS repository you selected.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:117]:
```suggestion
1. **(VCS Only)** Optionally specify the **VCS branch** within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses your selected VCS repository's default branch.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 9]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
+1. **(VCS Only)** Specify the **Policies path** in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:118]: Suggesting shorteding this for brevity and readability:
```suggestion
1. **(VCS Only)** Specify where your policy set files live using the **Policies path**. This lets you maintain multiple policy sets within a single repository. Use a relative path from your root directory to the directory that contains either the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration files. If you do not set this field, Terraform Cloud uses the repository's root directory.
```

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 10]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
+1. **(Managed Policies Only)** Select managed **Policies** to add to the policy set. You can only add policies written with the policy framework you selected for the policy set.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:119]: Does the UI say "Managed Policies"? Or just "Policies"? 
```suggestion
1. **(Managed Policies Only)** Select managed **Policies** to add to the policy set. You can only add policies written with the same policy framework you selected for this set.
```

**Review by rkoron007 — COMMENTED** [review]
Looks great to me! I added a few suggestions for style; but please feel free to push back on anything that doesn't feel good!

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx (hunk 9 above)
Comment (ker-an) [reply website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:118]: Love it!

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx [hunk 11]
```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
...
-   - **(VCS Only) Policies Path**: Specify the sub-directory in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.
-   - **(Managed Policies Only) Policies**: Select managed policies to add to the policy set. You can only add policies written with the policy framework you selected for the policy set.
+1. Choose a **Policy framework** for the policies you want to add. A policy set can only contain policies that use the same framework (OPA or Sentinel). You cannot change the framework type after creation.
```
Comment (rkoron007) [inline website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx:112]: I think this looks a little bit better if it's even more precise:
```suggestion
1. Choose a **Policy framework** for the policies you want to add. A policy set can only contain policies that use the same framework (OPA or Sentinel). You cannot change a policy set's framework type after creation.
```

**Review by rkoron007 — APPROVED** [review]
Teeny suggestion for style, but these look great to me! ✨

File: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx (hunk 11 above)
Comment (hs26gill) [reply website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx]: ✅

**Pull request merged**